### DISCUSSION TITLE
SunbirdED -Portal :: Question set is not displaying for content creator role

### DISCUSSION THREAD

**Discussion by princegupta1131** [Sunbird-coKreat/Community#85] [Bugs]
**In Sunbird Ed - Portal Question Set module is not visible for the content creator role but copy question set option is visible for the content creator.** 
![image](https://github.com/Sunbird-coKreat/Community/assets/114015020/08ffed3a-d7ae-410b-945e-f40dd2addbf6)

![image](https://github.com/Sunbird-coKreat/Community/assets/114015020/818037a9-6ad1-403b-b1cb-09f5bdfb3b19)

**For the Book creator role question set module is visible but a copy  question set option is not there**
<img width="1792" alt="image" src="https://github.com/Sunbird-coKreat/Community/assets/114015020/c3f89ea7-b9f3-40db-8eda-537568f02630">
![image](https://github.com/Sunbird-coKreat/Community/assets/114015020/7aa24661-0220-44a8-a6af-56c6ef99cd47)

can anyone please help on this ?

CC: @vinukumar-vs @rajeevsathish @Krishnaj20 @arun1205

**Comment by arun1205** [comment]
@princegupta1131 this looks like a form configuration problem.. Will look into this and update

**Reply by princegupta1131** [reply]
Hi @arun1205, @shubhambansaltarento this Issue got resolved by doing the code changes in the portal code base.
 Please refer to this PR for reference -> https://github.com/Sunbird-Ed/SunbirdEd-portal/pull/8738

**Comment by mmanimegalai** [comment]
@princegupta1131 Thanks for letting us know about the fix. This will be verified and merged as needed cc @vinukumar-vs
